frontend/dockerfile/cmd/dockerfile-frontend/hack/release

```diff
@@ -60,6 +60,11 @@ TAG=$2
 REPO=$3
 PUSH=$4
 
+platformFlag="--platform local"
```

Comment:
Shellcheck would complain against this (not enabled atm). It wants you to use `platformFlag="--platform=local"` in here and quote on usage `"$platformFlag"`.

Don't try to fix the rest of this script to pass shellcheck as part of this PR.